fe/fe-core/src/main/java/com/starrocks/qe/scheduler/dag/FragmentInstanceExecState.java

```diff
@@ -173,7 +173,7 @@ public void deployAsync() {
 
         TNetworkAddress brpcAddress = worker.getBrpcAddress();
         try {
-            if (serializedRequest.length != 0) {
+            if (serializedRequest != null && serializedRequest.length != 0) {
```

Comment:
Could you add some comments to explain in which case the serializedRequest is nullptr?

Reply:
when set enable_plan_serialize_concurrently = false that the default value is true or serialize encountered exception.